### PR TITLE
Add script covering case when VR.PerformInteraction gets UNSUPPORTED_…

### PR DESCRIPTION
**Pull request by vvvakulenko**
…RESOURCE and UI.PerformInteraction gets WARNINGS from HMI
Script is a part of TS #666

### CODE REVIEW

**Comment by vvvakulenko** [comment]
@istoimenova , Please review

File: test_scripts/API/PerformInteraction/020_ATF_VR_IsReady_no_resp_PerfInteract_VR_PerfInterac_UNSUP_RES.lua [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        EXPECT_HMICALL("BasicCommunication.ActivateApp")
+        :Do(function(_,data1)
+          self.hmiConnection:SendResponse(data1.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+        end)
+      end)
+    end
+  end)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
```
Comment (istoimenova) [inline test_scripts/API/PerformInteraction/020_ATF_VR_IsReady_no_resp_PerfInteract_VR_PerfInterac_UNSUP_RES.lua]: @vvvakulenko, as precondition for CreateIntercationChoiceSet, you should have icon.png on DUT storage. You can use as example: 
commonSteps:PutFile("Precondition_PutFile", "icon.png")

Comment (vvvakulenko) [inline test_scripts/API/PerformInteraction/020_ATF_VR_IsReady_no_resp_PerfInteract_VR_PerfInterac_UNSUP_RES.lua]: @istoimenova , updated on commit 3f8e3dccb1dd93d987943dceb961651a2e48676d

**Comment by vvvakulenko** [comment]
@istoimenova , Please check updated

**Pull request merged**